libs/experimental/langchain_experimental/sql/base.py

```diff
@@ -119,6 +119,8 @@ def _call(
                 callbacks=_run_manager.get_child(),
                 **llm_inputs,
             ).strip()
+            import re
```

Comment:
`re` can be pulled to global namespace

Would you be willing to create a helper function to clean up sql_cmd and a unit test for the clean up logic? it's common for the clean up logic to expand over time

Reply:
where is the place to put the unit test, it has the depended on the connection to DB, could you give an example we already have.

Reply:
Thanks, I already update the code as your suggestion.